src/components/button/button.scss

```diff
@@ -305,13 +307,14 @@ span {
     border-color: var(--kd-color-border-button-secondary-state-hover);
   }
 
-  &:active {
-    border-color: var(--kd-color-border-button-secondary-state-pressed);
-  }
-
   &:focus {
     border-width: 2px;
```

Comment:
I don't like how this causes the outline button size to change when focused. I think we should maintain the default border width, and just use outline for focus.

Reply:
Right, I had the same thoughts as well. I've added a comment in the Figma file to address this issue.

Reply:
Hi @jaideep08, are we waiting for this response? is it ok to track this as separate issue and merge the Tertiary button color issue.?